lib/factory_bot/find_definitions.rb

```diff
@@ -5,21 +5,27 @@ class << self
     # "factories", "test/factories" and "spec/factories". Only the first
     # existing file will be loaded.
     attr_accessor :definition_file_paths
-  end
 
-  self.definition_file_paths = %w[factories test/factories spec/factories]
+    def find_definitions
```

Comment:
This is a larger diff than I was expecting for this change. Would adding `load(path) if File.file?(path)` to the existing code be enough to get the test passing?

Reply:
You got it right, this was just a refactoring to make it look more linear, I'll drop the commit from this PR 👍

Reply:
Oh, now I remember, that was also in response to CodeClimate flagging that method for increased complexity, I still have the commit locally, let me know if you want me to add it back.

Reply:
What do you think about extracting a helper method instead. Something like:

```ruby
def load_file_or_directory(path)
  load("#{path}.rb") if File.exist?("#{path}.rb")
  
  load path if File.file? path

  if File.directory? path
    Dir[File.join(path, "**", "*.rb")].sort.each do |file|
      load file
    end
  end
end
```

Reply:
Good call! That was enough for CodeClimate ✅ 

One thing I just realized was in the refactoring is that there was a protection against loading a file twice (say you have `foo` and `foo.rb` in the paths). Honestly I don't remember if it was something we encountered on some solidus extension or it was just a side-effect of the refactoring.

You think would be worth adding a spec for that and fixing it in a separate PR?